apis/core/v1alpha2/featureflagconfiguration_conversion.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha2
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/open-feature/open-feature-operator/apis/core/v1alpha1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/conversion"
+)
+
+func (ffc *FeatureFlagConfiguration) SetupWebhookWithManager(mgr ctrl.Manager) error {
+	return ctrl.NewWebhookManagedBy(mgr).
+		For(ffc).
+		Complete()
+}
+
+func (src *FeatureFlagConfiguration) ConvertTo(dstRaw conversion.Hub) error {
+	dst := dstRaw.(*v1alpha1.FeatureFlagConfiguration)
+
+	dst.ObjectMeta = src.ObjectMeta
+	if src.Spec.ServiceProvider != nil {
```

Comment:
Do we have to check that `.Spec` is not `nil`, or will existing validation handle that?

Reply:
`.Spec` isn't a pointer type so a nil check isn't needed